pagefaultgames/pokerogue
# [Feature][ENV] add .env.test

## What are the changes?


add `.env.test` file

## Why am I doing these changes?




to be able to configure env variables for tests only

## What did change?


i18n is not spamming it's debug output in tests anymore.

### Screenshots/Videos



## How to test the changes?





run the tests (locally) and check if you still see this i18n debug abomination output. YOu shouldn't

## Checklist
